connection/connection.go

```diff
@@ -33,23 +33,23 @@ type CipherscanCiphersuite struct {
 //the following structs represent the output we want to provide to DB.
 
 type Stored struct {
-	ScanIP             string                       `json:"scanIP"`
-	FirstSeenTimestamp string                       `json:"firstSeenTimestamp"`
-	LastSeenTimestamp  string                       `json:"lastSeenTimestamp"`
-	ServerSide         bool                         `json:"serverside"`
-	CipherSuites       map[string]StoredCiphersuite `json:"cipherscanCiphersuite"`
-	CurvesFallback     bool                         `json:"curvesFallback"`
+	ScanIP             string                       `json:"scanIP,omitempty"`
+	FirstSeenTimestamp string                       `json:"firstSeenTimestamp,omitempty"`
+	LastSeenTimestamp  string                       `json:"lastSeenTimestamp,omitempty"`
+	ServerSide         bool                         `json:"serverside,omitempty"`
+	CipherSuites       map[string]StoredCiphersuite `json:"cipherscanCiphersuite,omitempty"`
```

Comment:
`CipherSuites` should be singular: `CipherSuite`
The json name shouldn't mention `cipherscan` at all, unless you're returning not just the ciphersuite, but the entire cipherscan json, in which case I would call this `cipherscanOutput` or something similar